src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/HttpsTests.cs

```diff
@@ -19,6 +19,7 @@ public class HttpsTests : ConditionalWcfTest
     // Server: BasicHttpsBinding default value is Soap11
     [ConditionalFact(nameof(Root_Certificate_Installed))]
     [OuterLoop]
+    [ActiveIssue(1123, PlatformID.AnyUnix)]
```

Comment:
Agree, but I would like @iamjasonp  to recommend a name for the condition and a proposed implementation for the detector.  Presumably it failed to write to the certificate store because it has not been configured as per https://github.com/dotnet/wcf/blob/master/Documentation/manualtest-guide.md .

If this is correct, I could make a condition "Certificate_Store_Configured" or something like that.  But how would the detector work?  And why wouldn't we add that to every single test requiring a root cert.  Thoughts welcome.

/cc @iamjasonp